x11-misc/mugshot/files/fix_env_spawn_args.patch

```diff
@@ -0,0 +1,47 @@
+--- a/mugshot_lib/SudoDialog.py	2018-04-12 00:23:21.000000000 +0200
```

Comment:
Are those patches accepted upstream? Include a link to their source.

Reply:
The `fix_env_spawn_args.patch` is written by myself. Without this one, Mugshot seems not to work on Gentoo, it doen's even start then. There is a ticket: https://bugs.launchpad.net/ubuntu/+source/mugshot/+bug/1443283. I've submited the patch there.

The two other patches are from Arch Linux. But it seems, upstream does not care about them.